crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1185,8 +1185,7 @@ impl PgManager {
         name_service_config: &NameServiceConfig,
         address: SuiAddress,
     ) -> Result<Option<String>, Error> {
-        let reverse_record_id =
-            name_service_config.reverse_record_field_id(NativeSuiAddress::from(address));
+        let reverse_record_id = name_service_config.reverse_record_field_id(address.as_slice());
```

Comment:
Why is this now a slice? The function takes a `SuiAddress`, and `address` is a `SuiAddress` so I am confused (without looking further into the actual types).

Reply:
this PR changes reverse_record_field_id to take &[u8]